src/cljs/athens/main/core.cljs

```diff
@@ -13,10 +13,13 @@
 (def log (js/require "electron-log"))
 
 
-;; NB: channels don't work on github releases, instead use use prereleases.
+;; Channels don't work on github releases, instead using prereleases.
+;; Also see .github/workflows/build.yml for more context and how auto-update data is published.
 ;; https://github.com/electron-userland/electron-builder/issues/1722#issuecomment-310468372
 ;; https://github.com/electron-userland/electron-builder/issues/4988
-(set! (.. autoUpdater -channel) "beta")
+;; https://www.electron.build/auto-update#appupdater-eventemitter
+(set! (.. autoUpdater -allowPrerelease) true)
```

Comment:
I though we've decided to not autoupdate prereleases.

Reply:
It won't be updated if there's no data to update it (above). But now that I think about it more, this part is redundant:

> allowPrerelease = false Boolean - GitHub provider only. Whether to allow update to pre-release versions. Defaults to true if application version contains prerelease components (e.g. 0.12.1-alpha.1, here alpha is a prerelease component), otherwise false.

The auto updater will set it to true if there's any prerelease components to the version, which is what I was doing anyway. Might as well just not do anything with it there.